### ISSUE TITLE
How do you download a blob link?Is there any Workaround?

### ISSUE THREAD

**Issue by adiltirur** [pichillilorenzo/flutter_inappwebview#481]
At the moment I saw your post in stackoverflow on how to initiate a download, but this is not possible for a blob file. Is there a workaround to download a blob file ?

**Comment by wpazio** [comment]
You can try convert blob url to base64 [Flutter WebView blob pdf download](https://stackoverflow.com/a/64902313/1631253)

**Comment by guoguoguilai** [comment]
Does it work in iOS?

**Comment by guoguoguilai** [comment]
I found a solution. Build a simple backend using Fluter with jaguar, then convert the blob as formData and post it to Flutter.

**Comment by ycv005** [comment]
@guoguoguilai Does converting any blob object to `formdata` and then downloading them is correct way ?

**Comment by risfat** [comment]
Did you find any working solutions?

**Comment by guoguoguilai** [comment]
> Did you find any working solutions?

Yes. You can check here: https://github.com/guoguoguilai/flutter-webview-blob-download

**Comment by RomanSoviak** [comment]
Any solutions?

**Comment by lucsoft** [comment]
still no good solution to it?

**Comment by EArminjon** [comment]
https://github.com/pichillilorenzo/flutter_inappwebview/issues/2212
